src/main/java/core/dao/StorageDao.java

```diff
@@ -0,0 +1,7 @@
+package core.dao;
+
+import java.util.Map;
+
+public interface StorageDao {
+    Map<String,Integer> getAll();
+}
```

Comment:
if you don't know what dao do, please, don't use it. Make your storage `public static final` and use it everywhere